pkgs/development/haskell-modules/non-hackage-packages.nix

```
@@ -47,5 +47,7 @@ self: super: {
   # Generated with:
   # nix-shell -I nixpkgs=$PWD -p cabal-install -p cabal2nix --run 'cabal update; cabal2nix cabal://arion-compose > pkgs/applications/virtualization/arion/arion-compose.nix'
   arion-compose = self.callPackage ../../applications/virtualization/arion/arion-compose.nix {};
+  # cabal2nix --revision <rev> https://gitlab.com/lysxia/ap-normalize.git
```

Rather use the following for generating the package which gets around the use of fetchgit (which is slow) and uses the sdist so we are closer to the hackage package.

```suggestion
  # cabal2nix cabal://ap-normalize-$version
```

---

fixed

---

Can you also regenerate with that command (and ideally a newer version of cabal2nix which you should be able to get from the darwin 21.05 channel)